Dockerfile

```diff
@@ -1,33 +1,61 @@
-FROM crops/yocto:ubuntu-16.04-base
+FROM ubuntu:16.04
 
 LABEL description="PELUX Yocto build environment"
 
-# Enables us to overwrite the user and group ID for the yoctouser. See below
-ARG userid=1000
-ARG groupid=1000
-
 USER root
 
 # Install dependencies in one command to avoid potential use of previous cache
 # like explained here: https://stackoverflow.com/a/37727984
-RUN apt-get update && apt-get install -y \
-        openssh-server \
-        iptables \
-        cvs \
+RUN apt-get update && \
+    apt-get install -y \
+        build-essential \
+        chrpath \
         coreutils \
-        python3-pip \
-        libfdt1 \
-        python-pysqlite2 \
+        cpio \
+        curl \
+        cvs \
+        debianutils \
+        diffstat \
+        g++-multilib \
+        gawk \
+        gcc-multilib \
+        git-core \
+        graphviz \
         help2man \
-        libxml2-utils \
+        iptables \
+        iputils-ping \
+        libegl1-mesa \
+        libfdt1 \
         libsdl1.2-dev \
-        graphviz \
+        libxml2-utils \
+        locales \
+        m4 \
+        openssh-server \
+        python \
+        python-pysqlite2 \
+        python3 \
+        python3-git \
+        python3-jinja2 \
+        python3-pexpect \
+        python3-pip \
         qemu-user \
-        g++-multilib \
-        curl \
         repo \
         rsync \
-        m4
+        screen \
+        socat \
+        subversion \
+        sudo \
+        sysstat \
+        texinfo \
+        tmux \
+        unzip \
+        wget \
+        xz-utils
+
+RUN apt-get clean
+
+# Remove all apt lists to avoid build caching
+RUN rm -rf /var/lib/apt/lists/*
```

Comment:
Why did you move this? Cleanup? Separate commit if "yes".

Reply:
It just made more sense for me that we cleanup cache once we're done with using `apt-get`.

Reply:
You are not done with apt-get at this point though. So not only is it wrong, it is hidden together with an unrelated change. Please make separate commits for things like this.